networkx/algorithms/centrality/voterank_alg.py

```diff
@@ -7,11 +7,9 @@ def voterank(G, number_of_nodes=None):
     """Select a list of influential nodes in a graph using VoteRank algorithm
 
     VoteRank [1]_ computes a ranking of the nodes in a graph G based on a
-    voting scheme. With VoteRank, all nodes vote for each of its in-neighbours
-    and the node with the highest votes is elected iteratively. The voting
-    ability of out-neighbors of elected nodes is decreased in subsequent turns.
-
-    Note: We treat each edge independently in case of multigraphs.
+    voting scheme. With VoteRank, every node votes for each of its neighbours
+    and the node with the highest votes is elected iteratively. If a node has been elected,
+    it doesn’t participate in subsequent voting, and the voting ability of its neighbors also be decreased in subsequent turns.
```

Comment:
I actually think the original wording here was more clear - the linked reference paper also describes things in the terms that were used originally (e.g. "in-neighbors" and "out-neighbors") and I think there's value in consistency there.

My (slight) preference would be to stick with the original wording.

Reply:
Thank you @rossbar , I made the suggested changes.